lib/matplotlib/backends/backend_qt5agg.py

```diff
@@ -48,10 +48,10 @@ def paintEvent(self, event):
             [[left, self.renderer.height - (top + height * self._dpi_ratio)],
              [left + width * self._dpi_ratio, self.renderer.height - top]])
         reg = self.copy_from_bbox(bbox)
-        buf = reg.to_string_argb()
+        # buf = reg.to_string_argb()
+        buf = memoryview(reg)
```

Comment:
I'm almost certain that this should not be part of this PR.

Reply:
Ha, no...